helpers/js.js

```diff
@@ -1,3 +1,5 @@
+"use strict";
```

Comment:
This lint ruleset feels familiar somehow.. Can't quite put my finger on it

:wink: